src/components/HostSettingsButton.tsx

```diff
@@ -106,17 +123,18 @@ export default function HostSettingsButton({
 												>
 													{storyPoint}
 												</label>
+												{allowedStoryPoints.includes(storyPoint) && (
```

Comment:
praise: technically and visually, this approach looks great to distinguish the current from the new ones ^.^

I didn't understand from the video your struggle/wish with this.

Reply:
Thanks!! I'm glad it works as is. 

I was thinking that when you open the dialog, the current points would be in the checked "green" state. But I ended up deciding to use the red dot, first below, then on the number. I like it better!!!